pkg/manager/member/ticdc_member_manager.go

```diff
@@ -126,8 +136,38 @@ func (tcmm *ticdcMemberManager) syncStatefulSet(tc *v1alpha1.TidbCluster) error
 	return updateStatefulSet(tcmm.stsControl, tc, newSts, oldSts)
 }
 
-// TODO syncTiCDCStatus
 func (tcmm *ticdcMemberManager) syncTiCDCStatus(tc *v1alpha1.TidbCluster, sts *apps.StatefulSet) error {
+	if sts == nil {
+		// skip if not created yet
+		return nil
+	}
+
+	tc.Status.TiCDC.StatefulSet = &sts.Status
+	upgrading, err := tcmm.ticdcStatefulSetIsUpgradingFn(tcmm.podLister, tcmm.pdControl, sts, tc)
+	if err != nil {
+		return err
+	}
+	if upgrading {
+		tc.Status.TiCDC.Phase = v1alpha1.UpgradePhase
+	} else {
+		tc.Status.TiCDC.Phase = v1alpha1.NormalPhase
+	}
+
+	ticdcCaptures := map[string]v1alpha1.TiCDCCapture{}
+	for id := range helper.GetPodOrdinals(tc.Status.TiCDC.StatefulSet.Replicas, sts) {
+		podName := fmt.Sprintf("%s-%d", controller.TiCDCMemberName(tc.GetName()), id)
+		capture, err := tcmm.cdcControl.GetStatus(tc, int32(id))
+		if err != nil {
+			return err
+		}
+		ticdcCaptures[podName] = v1alpha1.TiCDCCapture{
+			PodName: podName,
```

Comment:
Why PodName is set in the values since it's already set as key?

Reply:
It is a redundancy attribute like TiDBFailureMember does.